packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -111,6 +111,32 @@ const TEMPLATE_SOURCE_NPM = 'npm'
 const TEMPLATE_SOURCE_BUNDLE = 'bundle'
 const DEFAULT_TEMPLATE_VERSION = 'latest'
 
+const EXTENSION_QUESTIONS = [
+    {
+        name: 'project.generationType',
+        message: 'Do you want to generate a project using extensions or using a template?',
+        type: 'list',
+        choices: [
+            {name: 'Generate a project using extensions', value: 'extensions'},
+            {name: 'Generate a project using a template', value: 'template'}
+        ]
+    },
+    {
+        name: 'project.extension',
+        message: 'Which extension do you want to install?',
+        type: 'list',
+        // TODO: Get the list of available extensions dynamically
+        choices: [{name: 'extension-sample', value: 'extension-sample'}],
+        when: (answers) => answers.project.generationType === 'extensions'
```

Comment:
Should this type be "checkbox" to allow the user to check more than a single application extension?

Reply:
This implies that where you are using `extension` we want to have `extensions`, or rather `appExtensions|applicationExtensions`